packages/curve-ui-kit/src/features/connect-wallet/ui/ConnectedWalletLabel.tsx

```diff
@@ -17,6 +18,8 @@ export const ConnectedWalletLabel: FunctionComponent<ConnectedWalletLabelProps>
   ...props
 }) => (
   <Button size="small" color="ghost" onClick={onDisconnectWallet} {...props}>
-    <AddressLabel value={walletAddress} />
+    <Box maxWidth={200} title={walletAddress}>
```

Comment:
This hardcoded width isn't great here. I think we should have it in a separate `AddressLabel` component so it can be reused in other places

Reply:
I've precisely removed the `AddressLabel` component because it was too simple too be warranted a component. Sometimes it's a header, sometimes just a span, sometimes it should be in a `<Box>` etc. Just using the `addressShort` should suffice.

But the box with maxWidth is useless regardless given the usage of `addressShort` so I removed it.